File: ordenadores/sol.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int cy;
    cin >> cy;
    int n;
    cin >> n;
    vector<string> update;
    vector<string> retire;
    while (n--) {
        int y;
        string id, ig;
        cin >> id >> ig >> y >> ig >> ig >> ig >> ig;
        if (y <= cy-3) retire.push_back(id);
        else update.push_back(id);
    }
    cout << "ACTUALIZAR\n";
    for (auto id : update) cout << id << '\n';
    cout << "RETIRAR\n";
    for (auto id : retire) cout << id << '\n';
}

File: ordenadores/sol.py
update = []
retire = []
cy = int(input())
n = int(input())
for i in range(n):
    a = input().split()
    y = int(a[2])
    id = a[0]
    if y <= cy-3: retire.append(id)
    else: update.append(id)
print("ACTUALIZAR")
for id in update: print(id)
print("RETIRAR")
for id in retire: print(id)
